pwa/src/components/configurationsExportButton/ConfigurationsExportButton.tsx

```diff
@@ -27,19 +27,23 @@ export default function ConfigurationsExportButton() {
         });
       })
       .catch((error) => {
-        throw new Error (error)
+        throw new Error(error)
```

Comment:
This should say what kind of error it is, like we do in other files.

E.g.: `throw new Error('GET export: ' + err)`